src/content/docs/usage/ide.mdx

```diff
@@ -325,6 +327,32 @@ Daytona allows you to connect to your Workspace using JetBrains WebStorm IDE.
 
     You can now use the `daytona code` command to automatically open the JetBrains WebStorm IDE already connected to your Workspace.
 
+## Cursor
+
+Daytona allows you to connect to your Workspace using Cursor IDE.
+
+1. Run the following command to set your default IDE to Cursor:
+
+   ```shell
+   daytona ide
+   ```
+
+    Upon running this command, Daytona will display a list of available IDEs, allowing you to select `Cursor` as your default choice.
+
+    ```text
+      Choose Your Default IDE
+
+      | Cursor
+    ```
+
+2. Select `Cursor` from the list presented. Upon selecting, Daytona will set it as the default IDE and display a confirmation message.
+
+    ```text
+    Default IDE: Cursor
+    ```
+
+    You can now use the `daytona code` command to automatically open the Cursor IDE already connected to your Workspace.
+
```

Comment:
@stefanicjuraj please test i think you need to manually install Cursor CLI tools before this works.